lib/whitehall/authority/rules/document_rules.rb

```diff
@@ -8,7 +8,11 @@ def initialize(actor, subject)
     end
 
     def can?(action)
-      actor.gds_editor? || actor.departmental_editor? || action == :create
+      if action == :republish
```

Comment:
I _think_ these rules map to model types. It might be more appropriate to place this in `MiscellaneousRules`

Reply:
Does mapping this rule to the `Document` model feel off here?

Reply:
I think so yeah if you look at `RULESMAP`. That to me means the actions performed on a Document.

Whereas, the `MiscellaneousRules` include other administrative task like the emergency banner, and sitewide settings

Reply:
We had a chat about this and felt that republishing a document is an action performed on a document (the updated at date will change, I think?). The miscellaneous rules felt a little more meta. Might be worth chatting about this one tomorrow in case you have more context on this though?

Reply:
Happy to be overruled on this.

For me, this would map directly to a DocumentsController. Ultimately, it doesn’t really matter though!

The updated_at won’t change with this operation on the Whitehall models, only the updated_at of the content item will change.

Reply:
We've decided to go for your suggestion here as we're not updating the Whitehall model.